doc/flatpak-metadata.xml

```diff
@@ -182,6 +182,14 @@
                                 Available since 1.15.6.
                             </para></listitem></varlistentry>
 
+                            <varlistentry><term><option>usb</option></term>
+                            <listitem><para>
+                                USB device bus
+                                (all device nodes below
+                                <filename>/dev/bus/usb</filename>).
+                                Available since 1.15.9.
```

Comment:
```suggestion
                                Available since 1.15.11.
```

if it gets merged before the next development release

Reply:
yes, I'll adjust this. There is also the doc PR afterwards.